Source/Charts/Charts/BarLineChartViewBase.swift

```diff
@@ -181,7 +181,7 @@ open class BarLineChartViewBase: ChartViewBase, BarLineScatterCandleBubbleChartD
 
         // execute all drawing commands
         drawGridBackground(context: context)
-        
+        renderer.drawData(context: context)
```

Comment:
if you pull this up before clip, you would get multiple UT failures.
You have to restrict the filling area between the lines, not the whole chart.